src/App.js

```diff
@@ -0,0 +1,28 @@
+import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import Details from './pages/Details';
+import SearchPage from './pages/SearchPage';
+import ErrorPage from './pages/ErrorPage';
+
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/details/:name/:lat/:lon" element={<Details />} />
+        <Route path="/search/:name" element={<SearchPage />} />
+        <Route path="*" element={<ErrorPage />} />
+      </Routes>
+    </Router>
```

Comment:
- [ ] Your page does not render a design anywhere near the requirements. Kindly review your components, and `CSS` to make sure you have a view similar to the [Design Guildelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) 👍 
- [ ] The next review will look at the whole work 

### View:
![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/50990925/a8f07da7-ed3a-462c-8408-1c97ebe876a8)